config/co2eq_parameters_lifecycle.json

```diff
@@ -6576,6 +6576,13 @@
             "value": 199.07165644246462
           }
         ]
+      },
+	  "TH": {
```

Comment:
Thanks for correcting the number, can you please link the spreadsheet that I linked before instead please? And add the override emission factor for co2eq_parameters_direct.json

Thanks

Reply:
Added in now with the latest commit